pkg/sbom/sbom.go

```diff
@@ -28,6 +28,7 @@ type ScanOptions struct {
 	Timeout          time.Duration
 	WaitAfter        time.Duration
 	Fast             bool
+	NoCache          bool
```

Comment:
I think a comment could be useful to explain that we need an exception for host scans.